src/yafowil/plone/autoform/events.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-from zope.component.interfaces import IObjectEvent
-from zope.component.interfaces import ObjectEvent
+from zope.interface.interfaces import IObjectEvent
+from zope.interface.interfaces import ObjectEvent
```

Comment:
this is already done here https://github.com/bluedynamics/yafowil.plone/pull/29/commits/1300269d35f658c5cdb895baf2a2319ace07e7a6